app/src/main/res/layout/activity_session_detail.xml

```diff
@@ -8,6 +8,8 @@
             android:id="@+id/content_view"
             android:layout_width="match_parent"
             android:layout_height="match_parent"
+            android:background="@color/white"
+            android:transitionName="@string/session_detail"
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Warning: Attribute transitionName is only used in API level 21 and higher (current min is 16) [UnusedAttribute]